sqlserver/check.py

```diff
@@ -84,6 +86,11 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         self.connections = {}
         self.failed_connections = {}
         self.instances_metrics = {}
+        self.connector = init_config.get('connector', 'adodbapi')
```

Comment:
I think it's only documented at the instance level though? Maybe just add the same doc at the init_config level?